lib/repositories/fades-repository.ts

```diff
@@ -99,8 +99,8 @@ export class V2FadesRepository extends BaseRedshiftRepository {
     const formattedResult = result.map((row) => {
       const formattedRow: V2FadesRowType = {
         fillerAddress: row[0].stringValue as string,
-        faded: Number(row[1].longValue as number),
-        postTimestamp: Number(row[2].longValue as number),
```

Comment:
this ordering has to match the one in the SQL query

Reply:
Can you add this as a comment?